Add 4 changelog entries for Team:Security (#3889)
elastic/docs-content: release-notes/elastic-cloud-serverless/index.md

* Removes the global empty state redirect [#237612]({{kib-pull}}237612)
* Replaces the existing document count chart with RED metrics [#236635]({{kib-pull}}236635)

> * Sends cross cluster api key signature as header [#135674](https://github.com/elastic/elasticsearch/pull/135674)

Reviewer: ```suggestion
* Sends cross cluster API key signature as header [#135674](https://github.com/elastic/elasticsearch/pull/135674)
```

* Removes the AI Assistant Settings privilege [#239144]({{kib-pull}}239144)
* Adds ingest pipeline processor template suggestions to the manual ingest pipeline processor editor [#236919]({{kib-pull}}236919)
* Adds the `kibana.alert.index_pattern` field to all alerts [#239450]({{kib-pull}}239450)

> * Validates Certificate Identity provided in Cross Cluster API Key Certificate [#136299](https://github.com/elastic/elasticsearch/pull/136299)

Reviewer 2: ```suggestion
* Validates certificate identity from cross cluster credentials [#136299](https://github.com/elastic/elasticsearch/pull/136299)
```

* Removes the default port the from interactive setup cluster address form, unless specified [#230582]({{kib-pull}}230582)
* Fixes positioning of the **Add rule** popover on the Role Mappings page [#231551]({{kib-pull}}231551)

> * [ExtraHop][Qualys GAV] - Fix Cannot execute ILM policy delete step [#132387](https://github.com/elastic/elasticsearch/pull/132387)

Reviewer: ```suggestion
* Fixes an ExtraHop Qualys GAV bug that prevented executing the ILM policy delete step [#132387](https://github.com/elastic/elasticsearch/pull/132387)
```

* Ensures mapped fields are remembered across simulations [#233799]({{kib-pull}}233799)
* Adds time series telemetry in xpack usage [#134214](https://github.com/elastic/elasticsearch/pull/134214)

> * Adds InMemoryClonedSecureSettings for keeping temporary secure settings loaded [#134349](https://github.com/elastic/elasticsearch/pull/134349)

Reviewer 2: ```suggestion
* Add `LoadedSecureSettings` for keeping temporary secure settings loaded [#134349](https://github.com/elastic/elasticsearch/pull/134349)
```

* Removes the default port the from interactive setup cluster address form, unless specified [#230582]({{kib-pull}}230582)
* Fixes positioning of the **Add rule** popover on the Role Mappings page [#231551]({{kib-pull}}231551)

> * Fixes an ExtraHop Qualys GAV bug that prevented executing the ILM policy delete step [#132387](https://github.com/elastic/elasticsearch/pull/132387)

Reviewer 2: ```suggestion
* Adds `manage`, `create_index`, `read`, `index`, `write`, `delete`, permission for third party agent indices `kibana_system` [#132387](https://github.com/elastic/elasticsearch/pull/132387)
```

* Fixes **Discard** button state change logic for toggles [#223493]({{kib-pull}}223493)
* Removes `originId` from connectors during rule import [#223454]({{kib-pull}}223454)

> * Fixes ILM deletion step error permissions for cloud security posture [#128634](https://github.com/elastic/elasticsearch/pull/128634)

Reviewer 2: Remove PR that doesn't have changelog:
```suggestion
```

* Allows you to customize the table tab [#218686](https://github.com/elastic/kibana/pull/218686)
* Enables keyboard navigation for the create annotations form [#217918](https://github.com/elastic/kibana/pull/217918)

> * [Security Solution] Add `read` index privileges to `kibana_system` role for Microsoft Defender integration indexes [#126803](https://github.com/elastic/elasticsearch/pull/126803)

Reviewer: ```suggestion
* Adds `read` index privileges to `kibana_system` role for Microsoft Defender integration indexes [#126803](https://github.com/elastic/elasticsearch/pull/126803)
```

* Implements SAML custom attributes support in the Identity Provider plugin [#128176](https://github.com/elastic/elasticsearch/pull/128176)
* Fixes unsupported privileges error message during role and API key creation [#128858](https://github.com/elastic/elasticsearch/pull/128858)

> * Fixes unsupported privileges error message during role and API key crea… [#128858](https://github.com/elastic/elasticsearch/pull/128858)

Reviewer 2: ```suggestion
* Fixes unsupported privileges error message during role and API key creation [#128858](https://github.com/elastic/elasticsearch/pull/128858)
```

* Adds transport version support for IDP_CUSTOM_SAML_ATTRIBUTES_ADDED_8_19 [#128798](https://github.com/elastic/elasticsearch/pull/128798)
* Grants `kibana_system` reserved role access to "all" privileges to `.adhoc.alerts*` and `.internal.adhoc.alerts*` indices [#127321](https://github.com/elastic/elasticsearch/pull/127321)

> * Checks TooComplex exception for HasPrivileges body [#128870](https://github.com/elastic/elasticsearch/pull/128870)

Reviewer 2: ```suggestion
* Checks `TooComplex` exception for `HasPrivileges` body [#128870](https://github.com/elastic/elasticsearch/pull/128870)
```

* Fixes regression that caused the cases actions to disappear from the detections engine alerts table bulk actions menu [#215111]({{kib-pull}}215111)
* Changes "Close project" to "Log out" in nav menu in serverless mode [#211463]({{kib-pull}}211463)
* Fixes search profiler index reset field when query is changed [#215420]({{kib-pull}}215420)
* Adds ``manage_failure_store`` index privilege

> * [Failure Store] Feature-flag failure store privileges

Reviewer 2: This PR says its " behind the feature flag", so I think it should be omitted.
```suggestion
```
